webapp/admin/views.py

```diff
@@ -44,346 +44,54 @@ def get_stores():
-        return _handle_error_list(api_response_error_list.errors)
-    except (StoreApiError, ApiError) as api_error:
-        return _handle_error(api_error)
-
-    pending_invites = []
-    expired_invites = []
-    revoked_invites = []
-
-    for invite in invites:
-        if invite["status"] == "Pending":
-            pending_invites.append(invite)
-
-        if invite["status"] == "Expired":
-            expired_invites.append(invite)
-
-        if invite["status"] == "Revoked":
-            revoked_invites.append(invite)
-
-    sorted_pending_invites = sorted(
-        pending_invites, key=lambda item: item["expiration-date"]
-    )
-
-    sorted_expired_invites = sorted(
-        expired_invites, key=lambda item: item["expiration-date"]
-    )
-
-    sorted_revoked_invites = sorted(
-        revoked_invites, key=lambda item: item["expiration-date"]
-    )
-
-    member = next(
-        (
-            item
-            for item in members
-            if item["email"] == flask.session["publisher"]["email"]
-        ),
-        None,
-    )
-
-    return flask.render_template(
-        "admin/invites.html",
-        stores=stores,
-        store=store,
-        pending_invites=sorted_pending_invites,
-        expired_invites=sorted_expired_invites,
-        revoked_invites=sorted_revoked_invites,
-        member=member,
-    )
-
-
-@admin.route("/admin/<store_id>/settings")
+@admin.route("/admin/store/<store_id>")
 @login_required
 def get_settings(store_id):
     try:
-        stores = admin_api.get_stores(flask.session)
         store = admin_api.get_store(flask.session, store_id)
-        members = admin_api.get_store_members(flask.session, store_id)
     except StoreApiResponseErrorList as api_response_error_list:
         return _handle_error_list(api_response_error_list.errors)
     except (StoreApiError, ApiError) as api_error:
         return _handle_error(api_error)
 
-    member = next(
-        (
-            item
-            for item in members
-            if item["email"] == flask.session["publisher"]["email"]
-        ),
-        None,
-    )
+    return jsonify(store)
 
-    return flask.render_template(
-        "admin/settings.html", stores=stores, store=store, member=member
-    )
 
-
-@admin.route("/admin/<store_id>/settings", methods=["POST"])
+@admin.route("/admin/store/<store_id>/settings", methods=["POST"])
 @login_required
 def post_settings(store_id):
     settings = {}
-    settings["private"] = not flask.request.form.get("is_public")
+    settings["private"] = json.loads(flask.request.form.get("private"))
     settings["manual-review-policy"] = flask.request.form.get(
         "manual-review-policy"
     )
 
+    res = {}
+
     try:
         admin_api.change_store_settings(flask.session, store_id, settings)
-        flask.flash("Changes saved", "positive")
+        res["msg"] = "Changes saved"
     except StoreApiResponseErrorList as api_response_error_list:
```

Comment:
I think python linting is complaining that `api_response_error_list` is not being used.